internal/testutils/jimmtest/jimm_mock.go

```diff
@@ -3,166 +3,28 @@
 package jimmtest
 
 import (
-	"context"
-	"time"
-
-	"github.com/go-macaroon-bakery/macaroon-bakery/v3/bakery"
 	"github.com/google/uuid"
-	"github.com/juju/juju/api/base"
-	jujuparams "github.com/juju/juju/rpc/params"
 	"github.com/juju/names/v5"
 
-	"github.com/canonical/jimm/v3/internal/common/pagination"
-	"github.com/canonical/jimm/v3/internal/db"
-	"github.com/canonical/jimm/v3/internal/dbmodel"
-	"github.com/canonical/jimm/v3/internal/errors"
 	"github.com/canonical/jimm/v3/internal/jimm"
-	jimmcreds "github.com/canonical/jimm/v3/internal/jimm/credentials"
-	"github.com/canonical/jimm/v3/internal/openfga"
 	"github.com/canonical/jimm/v3/internal/pubsub"
-	"github.com/canonical/jimm/v3/internal/testutils/jimmtest/mocks"
 )
 
 // JIMM is a default implementation of the jujuapi.JIMM interface. Every method
 // has a corresponding funcion field. Whenever the method is called it
 // will delegate to the requested funcion or if the funcion is nil return
 // a NotImplemented error.
 type JIMM struct {
-	mocks.ControllerService
-	mocks.ModelManager
-
 	AuditLogManager_       func() jimm.AuditLogManager
 	GroupManager_          func() jimm.GroupManager
 	IdentityManager_       func() jimm.IdentityManager
 	LoginManager_          func() jimm.LoginManager
 	RoleManager_           func() jimm.RoleManager
 	PermissionManager_     func() jimm.PermissionManager
 	ServiceAccountManager_ func() jimm.ServiceAccountManager
-
-	AddAuditLogEntry_                  func(ale *dbmodel.AuditLogEntry)
```

Comment:
These mocks have been moved to `internal/testutils/jimmtest/mocks/jimm_juju_mock.go`